apps/client/src/app/pages/portfolio/activities/import-activities-dialog/import-activities-dialog.scss

```diff
@@ -17,4 +17,16 @@
       }
     }
   }
+
+  .mat-stepper-horizontal {
+    margin-top: 8px;
+  }
+
+  .mat-mdc-form-field {
+    margin-top: 16px;
+  }
+
+  ::ng-deep .mat-horizontal-stepper-header {
+    pointer-events: none !important;
```

Comment:
There is a `linear` mode to achieve this, no need to override the css. See this [example](https://material.angular.io/components/stepper/overview#stepper-variants).